packages/core-graphql/src/repositories/transactions.ts

```diff
@@ -255,6 +256,7 @@ class TransactionsRepository extends Repository {
      * @return {Object}
      */
     public async getFeeStatistics() {
+        const minFeeBroadcast = localConfig.get("dynamicFees.minFeeBroadcast");
```

Comment:
This should be resolved from the container, those config objects are only for plugin internal use.